README.md

```diff
@@ -55,26 +80,83 @@ cosign verify --key cosign.pub ghcr.io/avisi-cloud/structurizr-site-generatr:<ve
 
 Replace `<version>` with the version you want to verify.
 
-## Example usage
+## Usage
 
 These examples use the [example workspace](docs/example) in this repository.
 
-### Start a development server
+Once installed, Structurizr Site Generatr is operated via your terminal by issuing commands. Each command is explained here:
+
+### Help
+
+To learn about available commands, or parameters for individual commands, call Structurizr Site Generatr with the `--help` operator.
 
 ```shell
-structurizr-site-generatr serve --workspace-file docs/example/workspace.dsl
+installed> structurizr-site-generatr --help
+
+   docker> docker run -it --rm ghcr.io/avisi-cloud/structurizr-site-generatr:1.0.9 --help
+
+Usage: structurizr-site-generatr options_list
+Subcommands:
+    serve - Start a development server
+    generate-site - Generate a site for the selected workspace.
+    version - Print version information
+
+Options:
+    --help, -h -> Usage info
 ```
 
-### Generate a site from a Git repository
+### Version
+
+To query the version of Structurizr Site Generatr installed / used.
 
 ```shell
-structurizr-site-generatr generate-site \
-    --git-url https://github.com/avisi-cloud/structurizr-site-generatr.git \
-    --workspace-file docs/example/workspace.dsl \
-    --branches main \
+installed> structurizr-site-generatr version
+
+   docker> docker run -it --rm ghcr.io/avisi-cloud/structurizr-site-generatr:1.0.9 version
+
+Structurizr Site Generatr v1.0.9
+```
+
+### Generate a website from a [C4 Workspace](https://github.com/structurizr/dsl)
```

Comment:
```suggestion
### Generate a website

#### From a [C4 Workspace](https://github.com/structurizr/dsl)
```

Reply:
I think that this way, the headings make it more clear that there's two ways of generating a website.

Reply:
There are 2 ways to generate a website?

To my understanding there's always a C4 Workspace DSL file as source / inpput to the `generate-site` operation.  However, you can either use the installed or docker "runtime" of this generator to generate a website, i.e. create the output.

If you can source the workspace from elsewhere, say C4 / Structurizr-flavoured PlantUML, then this approach to headings applies.

Reply:
Ah, I see what you mean. What I meant to say was that you can source the C4 Workspace from either your local filesystem or from a remote Git repository. That's what I'd like to make clear in the headings.